s3/proto/s3.proto

```diff
@@ -55,6 +55,74 @@ service S3 {
 	rpc HeadBucket(BaseRequest) returns (Bucket) {}
 }
 
+message ListObjectPartsRequest {
+	string bucketName = 1;
+	string objectKey = 2;
+	string encodingType = 3;
+	string uploadId = 4;
+	int32 maxParts = 5;
+	int32 partNumberMarker = 6;
+}
+
+message ListObjectPartsResponse {
+	int32 errorCode = 1;
+}
+
+message AbortMultipartRequest {
+	string bucketName = 1;
+	string objectKey = 2;
+	string uploadId = 3;
+}
+
+message CompletePart {
+	int64 partNumber = 1;
+	string eTag = 2;
+}
+
+message CompleteMultipartRequest {
+	string bucketName = 1;
+	string objectKey = 2;
+	string uploadId = 3;
+	repeated CompletePart completeParts = 4;
+}
+
+message CompleteMultipartResponse {
+	int32 errorCode = 1;
+	string uploadID = 2;
+	string eTag = 3;
+}
+
+message InitMultiPartRequest {
+	string bucketName = 1;
+	string objectKey = 2;
+	Acl acl = 3;
+	uint32 storageClass = 4;
+	map<string, string> attrs = 5;

```

Comment:
Do we support to choose a specific backend when init multipart upload?

Reply:
@sfzeng ,  I have not support the feature in the PR yet. I can new a issue to track it